packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -34,34 +31,10 @@ export class BlockList extends Component {
 		this.renderItem = this.renderItem.bind( this );
 		this.renderAddBlockSeparator = this.renderAddBlockSeparator.bind( this );
 		this.renderBlockListFooter = this.renderBlockListFooter.bind( this );
-		this.shouldFlatListPreventAutomaticScroll = this.shouldFlatListPreventAutomaticScroll.bind( this );
 		this.renderDefaultBlockAppender = this.renderDefaultBlockAppender.bind( this );
-		this.onBlockTypeSelected = this.onBlockTypeSelected.bind( this );
-		this.keyboardDidShow = this.keyboardDidShow.bind( this );
-		this.keyboardDidHide = this.keyboardDidHide.bind( this );
 		this.onCaretVerticalPositionChange = this.onCaretVerticalPositionChange.bind( this );
 		this.scrollViewInnerRef = this.scrollViewInnerRef.bind( this );
 		this.getNewBlockInsertionIndex = this.getNewBlockInsertionIndex.bind( this );
-
-		this.state = {
-			blockTypePickerVisible: false,
-			isKeyboardVisible: false,
-		};
-	}
-
-	// TODO: in the near future this will likely be changed to onShowBlockTypePicker and bound to this.props
-	// once we move the action to the toolbar
-	showBlockTypePicker( show ) {
-		this.setState( { blockTypePickerVisible: show } );
-	}
-
-	onBlockTypeSelected( itemValue ) {
-		this.setState( { blockTypePickerVisible: false } );
-
-		// create an empty block of the selected type
-		const newBlock = createBlock( itemValue );
-
-		this.finishInsertingOrReplacingBlock( newBlock );
 	}
 
 	finishInsertingOrReplacingBlock( newBlock ) {
```

Comment:
This is be removed in a follow up PR, as we now have this logic in `InserterMenu`